lib/__testfixtures__/window.input.js

```diff
@@ -4,3 +4,5 @@ setTimeout(() => {
 const f = parseFloat('5.5');
 const a = window.closed;
 window.status = 'open';
+status = 'closed';
+const s = status;
```

Comment:
Can you test the getter case in trickier positions? E.g. passing to a function, or `navigator.getBattery()` (should translate to Window_navigator_get).